packages/nimble-tokens/dist/fonts/css/fonts.css

```diff
@@ -5,3 +5,19 @@
   src: url('../assets/SourceSansPro-Regular.woff2') format('woff2');
   font-display: swap;
 }
+
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
```

Comment:
It looks like the site also defines different font files at these levels for italics. Italic fonts seem to be in [workspace selector](https://dev.azure.com/ni/DevCentral/_git/Skyline?path=/Web/Workspaces/SystemLinkShared/projects/systemlink-lib-angular/auth/src/sl-workspace-selector/sl-workspace-selector.component.scss&version=GBmaster&line=18&lineEnd=19&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) and some [custom property list items](https://dev.azure.com/ni/DevCentral/_git/Skyline?path=/Web/Workspaces/ExecutionManager/src/app/executions-editor/readonly-properties-list/readonly-properties-list.component.scss&version=GBmaster&line=25&lineEnd=26&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents).

Do we know if these usages are rendering in italics "good enough" cross-browser to avoid bringing in the italic font asset file? Or maybe we should bring them in to help get better consistency?

If the browser rendering is "good enough" I might lean away from adding more assets to improve page loading but that doesn't come with any benchmarking or real-world testing to see if that is actually a benefit. It's possible that because we use font-display: swap it's not actually a big deal.

Reply:
From what I can see, the italics look pretty good. The exception is semi-bold + italics (used in group headers within the table), but pulling in the semi-bold font makes that look much better.

I'm inclined to say we should wait to pull in the italic fonts, but I don't feel too strongly. I'm just not sure where we would want to draw the line of fonts to include since there are italic versions of each font-weight.

Reply:
The sanity check sounds good to me, I don't have any other concerns there with relying on the browsers font italization 👍